File: Web/API_Database/Controllers/EmployeeDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API_Database.Controllers
{
    public class EmployeeDTO
    {
        public string IdEmp { get; set; }
        public string FullNameEmp { get; set; }
        public string Roles { get; set; }
        public Boolean IsActive { get; set; }
        public string Sex { get; set; }
        public string CCCD { get; set; }
        public string DoB { get; set; }
        public string AddressEmp { get; set; }
        public string Phone { get; set; }
        public string password { get; set; }
        public string Picture { get; set; }
        public string Email { get; set; }
        public String TimeCheckIn { get; set; }
        public String TimeCheckOut { get; set; }
        public String Total_working_hours { get; set; }

    }
}

File: Web/MiniStoreWebMoblie/src/main/java/com/ministore/web/DTO/EmployeeDTO.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.ministore.web.DTO;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import java.sql.Date;

/**
 *
 * @author daigi
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class EmployeeDTO {
    private String idEmp;
    private String fullname;
    private boolean sex;
    private String cccd;
    private Date dob;
    private String address;
    private String phone;
    private Date datejoin;
    private String username;
    private String password;
    private int roles;
    private boolean isactive; 
    private String picture;
    private int snpt;
}
